openstack/sap-seeds/templates/flavor-seed.yaml

```diff
@@ -38,6 +38,16 @@ spec:
       "hw_video:ram_max_mb": "16"
       "catalog:alias": "m1.small,m1.smallhdd"
       "trait:CUSTOM_HANA_EXCLUSIVE_HOST": "forbidden"
+  - name: "c_c2_m2_cc3test"
```

Comment:
I think `hana_cc3test` would be a better name, as this is what it's supposed to be used for. Thus, it should live at the end of the file and get a new id.

We should also add a comment (in the next line?) to very shortly say why we need this flavor.

Reply:
flavor name changes to "hana_cc3test" and moved to end of file with comment why its needed.